### DISCUSSION TITLE
"When using Milvus to construct a vector database, if the transformer model is changed, the vector values for the text would also change. In that case, do I need to delete and re-create the database?"

### DISCUSSION THREAD

**Discussion by YongMinMax** [milvus-io/milvus#25945] [Q&A and General discussion] [answered]
"When using Milvus to construct a vector database, if the transformer model is changed, the vector values for the text would also change. To perform similarity search, do I need to delete and re-create the database?"

**Comment by yhmo** [comment, marked as answer]
Vectors come from different transformer models should not be put into the same collection.
For example:
create collection_1 to store vectors from model_1
create collection_2 to store vectors from model_2

**Reply by YongMinMax** [reply]
"Is it not possible to insert vectors from different models into one collection?"

**Reply by yhmo** [reply]
You can insert vectors from different models into one collection, but you may get unexpected search results.

**Reply by YongMinMax** [reply]
Thank you!
